en/docs/reference/connectors/creatio-connector/creatio-connector-example.md

```diff
@@ -0,0 +1,360 @@
+# Creatio Connector Example
+
+## What you’ll build 
+
+This example explains how to use the Creatio connector to connect with a CreatioCRM instance and perform the following operations.
+
+- Create Contact
+- Retrieve information of the created Contact
```

Comment:
Having "Contract" starting with caps is not correct. We need to have title case for the entire sentence or Sentence case. This change needs to match the other bullet points.